### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/tutorial/query-param-models.md`

### PR DESCRIPTION
**Pull request by timothy-jeong**

### CODE REVIEW

**Comment by jts8257** [comment]
Hi @alejsdev ,

Thank you for reviewing my PR. I noticed that the PR is labeled with `lang-zh`, but it actually contains a Korean translation (`lang-ko`). Could you kindly update the label to `lang-ko` so it correctly reflects the content of the PR?

**Comment by alejsdev** [comment]
My bad! Fixed.
Thanks @jts8257 :raised_hands:

File: docs/ko/docs/tutorial/query-param-models.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# 쿼리 매개변수와 모델
+
+연관된 쿼리 **매개변수** 그룹이 있다면 **Pydantic 모델** 을 사용해 선언할 수 있습니다.
+
+이렇게 하면 **여러 곳**에서 **모델을 재사용**할 수 있을 뿐만 아니라, 매개변수에 대한 검증 및 메타데이터도 한 번에 선언할 수 있습니다. 😎
+
+/// note | 참고
+
+이 기능은 FAST API 버전 `0.115.0`부터 제공됩니다. 🤓
```
Comment (hard-coders) [inline docs/ko/docs/tutorial/query-param-models.md:9]:
```suggestion
이 기능은 FastAPI 버전 `0.115.0`부터 제공됩니다. 🤓
```

File: docs/ko/docs/tutorial/query-param-models.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# 쿼리 매개변수와 모델
```
Comment (hard-coders) [inline docs/ko/docs/tutorial/query-param-models.md:1]:
```suggestion
# 쿼리 매개변수 모델
```
여기서는 새로운 기능인 매개변수 모델 자체를 의미하므로 "와"는 뻈으면 좋을거 같습니다.

**Review by hard-coders — CHANGES_REQUESTED** [review]
Great job! I reviewed only two points.

**Comment by jts8257** [comment]
Thank you for promptly updating the label to `lang-ko`, @alejsdev . I appreciate your swift action! 🙏

**Comment by jts8257** [comment]
Thank you for your valuable feedback, @hard-coders. 
I've addressed the two points you mentioned. Please let me know if there's anything else I should consider.

**Review by hard-coders — APPROVED** [review]
LGTM 🚀

**Comment by alejsdev** [comment]
Thanks for your contribution! @jts8257 :nerd_face: 
And thanks for your review! @hard-coders :rocket:

**Pull request merged**